src/container-mapping.ts

```diff
@@ -85,6 +85,24 @@ export class ContainerMapping implements IMicrosoftSecurityDevOps {
             dockerEvents: [],
             dockerImages: []
         };
+       
+        let bearerToken: string | void = await core.getIDToken()
+            .then((token) => { return token; })
+            .catch((error) => {
+                throw new Error("Unable to get token: " + error);
+            });
+
+        if (!bearerToken) {

```

Comment:
Might be useful to trim the string and ensure it doesn't only contain empty whitespaces.

Reply:
The logs I've seen don't indicate this has ever happened. The endpoint will return either the token or will result in an error.